WikiDrips/WikiTableViewController.swift

```diff
@@ -161,8 +174,10 @@ class WikiTableViewController: UITableViewController {
 // MARK: - UISearchResultsUpdating
 extension WikiTableViewController: UISearchResultsUpdating {
     func updateSearchResults(for: UISearchController) {
-        guard let searchText = searchController.searchBar.text, searchText.characters.count > 2 else { return }
-        search(for: searchText)
+        if let text = searchController.searchBar.text, text.characters.count > 2 {
```

Comment:
If we have coalescing I think we should remove the character limit. I also find it (as a user) limiting that I can't search for short words (i.e. like Pi)